README.md

```diff
@@ -26,9 +26,9 @@ Note: when starting a new appeal as an appellant, if you cannot progress beyond
 Install NodeJS using [nvm](https://github.com/nvm-sh/nvm#installing-and-updating)
 
 ```
-nvm install 18
-nvm use 18
-nvm alias default 18
+nvm install 20.17.0
+nvm use 20.17.0
```

Comment:
Perhaps leave for now, but I think you can just do `nvm install 20` to get the latest minor version?